pubspec.yaml

```diff
@@ -4,11 +4,12 @@ version: 1.3.2
 homepage: https://github.com/xantiagoma/dart_date
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=3.0.2 <4.0.0"
+  flutter: ">=1.17.0"
 
 dependencies:
-  intl: ">=0.17.0 <0.19.0"
```

Comment:
If nothing has changed in this package, I would recommend just removing or bumping the upper bound so that people using older versions of intl can keep using dart_date.

I also don't think it's necessary to bump the requirement for `timeago`, right?